nextcloud-startup-script.sh

```diff
@@ -217,6 +217,7 @@ download_script MENU menu
 download_script MENU server_configuration
 download_script MENU nextcloud_configuration
 download_script MENU additional_apps
+download_script NOT_SUPPORTED not-supported
```

Comment:
> I'm fine with everything except putting the script in the released version: `nextcloud-startup-script.sh`

Do you refer to this line?

Reply:
Yes.

Reply:
okay, I've removed it from the startup script